project/BuildHelper.scala

```diff
@@ -71,25 +83,41 @@ object BuildHelper {
   def stdSettings(prjName: String) = Seq(
     name := s"$prjName",
     scalacOptions := stdOptions,
-    crossScalaVersions := Seq("2.13.2", "2.12.11"),
+    crossScalaVersions := Seq(Scala213, Scala212, Scala3),
     scalaVersion in ThisBuild := crossScalaVersions.value.head,
     scalacOptions := stdOptions ++ extraOptions(scalaVersion.value),
-    libraryDependencies ++= compileOnlyDeps ++ testDeps ++ Seq(
-      compilerPlugin("org.typelevel"   % "kind-projector"  % "0.11.0") cross CrossVersion.full,
-      compilerPlugin("com.github.ghik" % "silencer-plugin" % "1.7.1") cross CrossVersion.full
-    ),
+    libraryDependencies ++= testDeps ++ {
+      if (isDotty.value)
+        Seq(
+          ("com.github.ghik" % s"silencer-lib_$Scala213" % silencerVersion % Provided)
```

Comment:
We may get rid of silencer now with Scala `2.12.13` - it fully supports `@nowarn` annotation